src/gui_common/CustomWindow.cs

```diff
@@ -16,6 +17,13 @@ public class CustomWindow : Control
 
     private bool hasBeenRemovedFromTree;
 
+#pragma warning disable CA2213
+    private Array<AddWindowReorderingSupportToSiblings> windowReorderingNodes = new();
```

Comment:
Is this necessary to use the Godot array type here? If it is not absolutely necessary (for use in Godot calls often etc.) List<T> should always be preferred in other situations.

Reply:
I will change it. It wasn't an intention to use it.